en/editor/extension/contributions-messages.md

```diff
@@ -28,37 +28,37 @@ In the top menu "Developer" - "Message List", the editor presets a message manag
 
 ### public 
 
-Type {string} Optional
+Type `{string}` Optional
 
 Whether to display this message externally, if true, the basic information of this message will be displayed on the message list interface.
 
 ### description
 
-Type {string} Optional
+Type `{string}` Optional
 
-If public is true, some simple descriptions will be displayed in the message list, supporting i18n:key syntax.
+If public is true, some simple descriptions will be displayed in the message list, supporting `i18n:key` syntax.
 
 ### doc
 
-Type {string} Optional
+Type `{string}` Optional
 
 If public is true, some documents of this message will be displayed, supporting i18n:key syntax.
```

Comment:
i18n: key...... (A space is missing after the colon)

Reply:
thanks. I didn't know this.